ngraph/test/CMakeLists.txt

```diff
@@ -414,6 +414,10 @@ target_link_libraries(unit-test PRIVATE ngraph_test_util
                                         ngraph::builder
                                         ngraph_backend
                                         libgtest)
+if (LINUX)
+    target_compile_options(unit-test PRIVATE -fno-lto)
+    target_link_options(unit-test PRIVATE -Wl,--exclude-libs,ALL -fno-lto)
```

Comment:
Is `--exclude-libs` needed if it is part of global settings: https://github.com/openvinotoolkit/openvino/blob/06145e20fc2e306018bc4fb3839810dd20b179c7/cmake/developer_package/compile_flags/os_flags.cmake#L302?

Reply:
it is, but for shared libraries. Here I do this for unit-test program

Reply:
right ;)